src/com/owncloud/android/db/UploadDbHandler.java

```diff
@@ -0,0 +1,416 @@
+        // cv.put("path", filepath);
+        // cv.put("account", account);
+        // cv.put("attempt",
+        // UploadStatus.UPLOAD_STATUS_UPLOAD_LATER.getValue());
+        // cv.put("message", message);
+        // long result = mDB.insert(TABLE_UPLOAD, null, cv);
+        // Log_OC.d(TABLE_UPLOAD, "putFileForLater returns with: " + result +
+        // " for file: " + filepath);
+        // return result != -1;
+    }
+
+    // ununsed until now. uncomment if needed.
+    // public Cursor getFailedFiles() {
+    // return mDB.query(TABLE_INSTANT_UPLOAD, null, "attempt>" +
+    // UploadStatus.UPLOAD_STATUS_UPLOAD_LATER, null, null, null, null);
+    // }
+
+    // ununsed until now. uncomment if needed.
+    // public void clearFiles() {
+    // mDB.delete(TABLE_INSTANT_UPLOAD, null, null);
+    // }
+
+    private class OpenerHelper extends SQLiteOpenHelper {
+        public OpenerHelper(Context context) {
+            super(context, mDatabaseName, null, mDatabaseVersion);
+        }
+
+        @Override
+        public void onCreate(SQLiteDatabase db) {
+            // PRIMARY KEY should always imply NOT NULL. Unfortunately, due to a
+            // bug in some early versions, this is not the case in SQLite.
+            db.execSQL("CREATE TABLE " + TABLE_UPLOAD + " (" + " path TEXT PRIMARY KEY NOT NULL UNIQUE,"
+                    + " uploadStatus INTEGER NOT NULL, uploadObject TEXT NOT NULL);");
+            // uploadStatus is used to easy filtering, it has precedence over
+            // uploadObject.getUploadStatus()
+        }
+
+        @Override
+        public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
+            if (newVersion == 4) {
+                db.execSQL("DROP TABLE IF EXISTS " + "instant_upload" + ";"); // drop
+                                                                              // old
+                                                                              // db
+                                                                              // (name)
+                db.execSQL("DROP TABLE IF EXISTS " + TABLE_UPLOAD + ";");
+                onCreate(db);
+            }
+
+        }
+    }
+
+    /**
+     * Stores an upload object in DB.
+     * 
+     * @param uploadObject
+     * @param message
+     * @return true on success.
+     */
+    public boolean storeUpload(UploadDbObject uploadObject) {
+        Log_OC.e(TAG, "Inserting "+uploadObject.getLocalPath()+" with uploadStatus="+uploadObject.getUploadStatus());
+        
+        ContentValues cv = new ContentValues();
+        cv.put("path", uploadObject.getLocalPath());
+        cv.put("uploadStatus", uploadObject.getUploadStatus().value);
+        cv.put("uploadObject", uploadObject.toString());
+
+        long result = getDB().insert(TABLE_UPLOAD, null, cv);
+        
+        Log_OC.d(TAG, "putFileForLater returns with: " + result + " for file: " + uploadObject.getLocalPath());
```

Comment:
Yes, that was the old name. Now the whole line can be removed as errors are logged below.